community-operators/svt-app-operator/svt-app-operator.v0.0.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: svt-app-operator.v0.0.8
+  namespace: placeholder
+  annotations:
```

Comment:
If there are example CRs for this operator, please provide them as a json list under `metadata.annotations.alm-examples`

Reply:
Done